src/test/components/__snapshots__/MarkerChart.test.js.snap

```diff
@@ -416,7 +416,7 @@ exports[`MarkerChart renders the hoveredItem markers properly 2`] = `
 <div
   class="tooltip"
   data-testid="tooltip"
-  style="left: 61px; top: 16px;"
+  style="left: 211px; top: 16px;"
```

Comment:
This comes from the change to `pageX` I did, as a result the tooltip is positioned further right.